File: app/src/main/java/com/project/scratchstudio/kith_andoid/Model/AnnouncementInfo.java
package com.project.scratchstudio.kith_andoid.Model;

import java.io.Serializable;

public class AnnouncementInfo implements Serializable {
    public int id;
    public String title;

    public int enabled;
    public int boardSubscription;
    public int subscriptionOnBoard;

    public String startDate;
    public String endDate;

    public String participants;
    public String needParticipants;
    public String url;
    public String description;

    public int organizerId;
    public String organizerName;
    public String organizerLastName;
    public String organizerMiddleName;
    public String organizerLogin;
    public String organizerPhoto;

}

File: app/src/main/java/com/project/scratchstudio/kith_andoid/model/AnnouncementInfo.kt
package com.project.scratchstudio.kith_andoid.model

import java.io.Serializable

class AnnouncementInfo : Serializable {
    var id: Int = 0
    lateinit var title: String

    var enabled: Int = 0
    var boardSubscription: Int = 0
    var subscriptionOnBoard: Int = 0

    lateinit var startDate: String
    lateinit var endDate: String

    lateinit var participants: String
    lateinit var needParticipants: String
    lateinit var url: String
    lateinit var description: String

    var organizerId: Int = 0
    lateinit var organizerName: String
    lateinit var organizerLastName: String
    lateinit var organizerMiddleName: String
    lateinit var organizerLogin: String
    lateinit var organizerPhoto: String
    lateinit var organizerPhone: String

}
